google/api_core/universe.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helpers for universe domain."""
+
+from google.auth.exceptions import MutualTLSChannelError
+from typing import Any, Optional
+
+DEFAULT_UNIVERSE = "googleapis.com"
+
+mTLS_Universe_Error = MutualTLSChannelError(
```

Comment:
Do we test for this anywhere? We should. Possibly in a separate PR if it's not relevant here....but then maybe we should omit this declaration, since it's not being used anywhere in this PR.

Reply:
addressed in f5bff28d6d2deda1a7316a065e3752cd0c5bed73.